packages/rpc/src/chainStorageWatcher.ts

```diff
@@ -71,14 +71,17 @@ export const makeAgoricChainStorageWatcher = (
   >();
 
   const watchedPathsToSubscribers = new Map<string, Set<Subscriber<unknown>>>();
-  const watchedPathsToRefreshTimeouts = new Map<string, number>();
+  const watchedPathsToRefreshTimeouts = new Map<
+    string,
+    ReturnType<typeof setTimeout>
```

Comment:
RE type change, `window.setTimeout` is `number`, but `globalThis.setTimeout` is `NodeJS.Timeout`. It was suggested [here](https://stackoverflow.com/a/50428377) to use type inference.